src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput frontBeamBreak;
+    private final DigitalInput backBeamBreak;
+
+    TalonFXConfiguration INTAKE_MOTOR_CONFIG = new TalonFXConfiguration();
```

Comment:
i would probably move this config to the apply configs method. There's no reason we need to access it outside of applying the configs that I can see